conftest.py

```diff
@@ -58,6 +58,17 @@ def zarr_store(tmpdir, request):
     return filepath
 
 
+@pytest.fixture()
+def zarr_store_scalar(tmpdir):
+    import zarr
+
+    # can/should we create a memorystore instead?
```

Comment:
yes, the bug should still show itself for a memorystore, because the bug is at the level of zarr keys, before the store implementation maps them to filepaths/memory locations.